src/VerifyPIVController.js

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint complexity:[2, 10], max-params: [2, 11] */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/FormType',
+  'views/shared/FooterWithBackLink',
+  'q'
+],
+function (Okta, FormController, FormType, FooterWithBackLink, Q) {
+
+  var _ = Okta._,
+      $ = Okta.$;
+  var { Util } = Okta.internal.util;
+
+  return FormController.extend({
+    className: 'mfa-verify verify-piv',
+    Model: {
+
+      save: function () {
+        this.trigger('request');
+        var self = this,
+            data = {
+              fromURI: this.settings.get('relayState')
+            },
+            pivButton = this.settings.get('piv');
+        return Q($.post({
```

Comment:
This post triggers the browser to prompt for cert ? I  was expecting to see us calling some browser api somewhere.

Reply:
Yea, the server needs to ask for the cert